cmd/image/import/import.go

```diff
@@ -252,6 +251,7 @@ func init() {
 	Cmd.Flags().StringVar(&pkg.ImageCMDOptions.AccessKey, "accesskey", "", "Cloud Object Storage HMAC access key.")
 	Cmd.Flags().StringVar(&pkg.ImageCMDOptions.SecretKey, "secretkey", "", "Cloud Object Storage HMAC secret key.")
 	Cmd.Flags().StringVar(&pkg.ImageCMDOptions.ImageName, "pvs-image-name", "", "Name to PowerVS imported image.")
+	Cmd.Flags().StringVar(&pkg.ImageCMDOptions.BucketAccess, "bucket-access", "", "Cloud Object Storage bucket access either public or private")
```

Comment:
wondering if we can use boolean type of arg because this value will always will be true or false, either public or not. e.g: `--public` flag and keep this as false by default.

Reply:
Also add the scenario in the help/example section

Reply:
Added a flag --public-bucket and updated the help section with importing from public bucket scenario.